ui/src/entities/User/index.ts

```diff
@@ -0,0 +1,110 @@
+import 'cross-fetch/polyfill';
```

Comment:
We don't need to do it in every entity. I think we should move this to a common place where it is guaranteed to be executed. e.g `pages/_app.tsx`.